include/sound/sof.h

```diff
@@ -57,6 +57,17 @@ enum sof_ipc_type {
 	SOF_IPC_TYPE_COUNT
 };
 
+struct sof_loadable_file_profile {
+	enum sof_ipc_type ipc_type;
+
+	const char *fw_path;
+	const char *fw_name;
+	const char *fw_lib_path;
+	const char *fw_path_postfix;
```

Comment:
We are adding a postfix to the firmware/library path in case of community key signed binaries.
This is to hold that postfix, but in a generic way, not tied to the word 'community'

Reply:
ah ok, probably a comment would help to explain that the same postfix will be used for path and lib.

Note that there are already exceptions to the rule, the fw and lib may be signed with different keys. Assuming that both share the same keys and use the same prefix as a consequence will not work in real life for Windows devices.

Reply:
> Note that there are already exceptions to the rule, the fw and lib may be signed with different keys. Assuming that both share the same keys and use the same prefix as a consequence will not work in real life for Windows devices.

:astonished:

Reply:
This hasn't changed, we need support for separate keys for fw and libs. Not my decision, it's how the hardware works and how Windows distributed firmware since SKL.